sky/resources.py

```diff
@@ -365,6 +365,20 @@ def _try_validate_accelerators(self) -> None:
             # because e.g., the instance may have 4 GPUs, while the task
             # specifies to use 1 GPU.
 
+        # Validate whether accelerator is available in specified region/zone.
+        acc, acc_count = list(acc_requested.items())[0]
+        if self.region is not None or self.zone is not None:
+            if not self._cloud.accelerator_in_region_or_zone(
+                    acc, acc_count, self.region, self.zone):
+                error_str = (f'Accelerator "{acc}" is not available in '
+                             '"{}" region/zone.')
```

Comment:
```suggestion
                             '"{}".')
```
Just a minor pref to remove some ambiguity.

Reply:
Fixed!